python/ray/train/BUILD

```diff
@@ -342,6 +342,14 @@ py_test(
     deps = [":train_lib", ":conftest"]
 )
 
+py_test(
+    name = "test_gpu_2",
+    size = "large",
```

Comment:
should it be a large?

Reply:
looks like it ran on CI in 95 seconds, so changed it to medium.